index.html

```diff
@@ -899,57 +574,512 @@ <h3><dfn>RTCQuicTransportState</dfn> Enum</h3>
       </div>
     </section>
   </section>
+  <section id="quic-transport*">
+    <h2><dfn>QuicTransport</dfn> Interface</h2>
+    <p>The <code>QuicTransport</code> is a subclass
+    of <code>QuicTransportBase</code> focused on client/server use
+    cases.</p>
+    <section id="quictransport-operation*">
+      <h3>Operation</h3>
+      <p>A <code><a>QuicTransport</a></code> instance is constructed
+      using an URL which is constrained by CORS, just like a WebSocket.</p>

```

Comment:
I know you're purposely leaving the CORS explanation out for now, but should we at least add a reference to CORS here?

Reply:
I added a different mechanism for CORS instead (a QUIC transport parameters for web origins).